### DISCUSSION TITLE
Build fails on '0.2.0' branch because of 'asciidoctor' configuration in 'pom.xml' file

### DISCUSSION THREAD

**Discussion by adrianbranescu** [Project-Books/book-project#851] [Q&A] [answered]
Issue is already solved on 'main' branch, but not on '0.2.0' branch.

I took 'pom.xml' configuration from 'main' branch and on '0.2.0' branch I managed to fix the build and run the automatic tests on my local computer.

Since I already have another issue assigned to me and I am relatively new to this project, I am only allowed to own one issue at a time. So, could someone else please fix this 'asciidoctor-maven-plugin' problem on '0.2.0'?

You can take the solution from here:
https://github.com/Project-Books/book-project/commit/a79e860e8f620f71a23ac348b71893ff260917eb

**Comment by knjk04** [comment, marked as answer]
Hi @adrianbranescu, we're not currently working on the 0.2.0 branch (even though we're still working towards that release).

I'll look to delete that branch to avoid confusion
